slash-commands/moderation/unban.js

```diff
@@ -0,0 +1,60 @@
+const {SlashCommandBuilder} = require('@discordjs/builders');
+const {promisePool} = require('../../config/db');
+const {sendToAuditLogsChannel} = require('../../helpers/sendToAuditLogs');
+const {
+  isServerStaff,
+  sendNoTargetStaffReply,
+} = require('../../helpers/validation');
+
+module.exports = {
+  data: new SlashCommandBuilder()
```

Comment:
I brought this up a couple of weeks ago but I don't think we reached a consensus. Should we add an optional field for the reason for the unban? This was something I implemented in the removetimeout command.

Reply:
I think we can leave this out from this PR for the moment (while we are focusing on converting the slashcommands) until there is more agreement that it is indeed needed. Then we can evaluate how these commands are working and go back later to add it needed. Basically, if we are not sure, I think it's better to not add the code.

Reply:
After checking with more mods it seems like it is a desired feature. I can add the reason to the PR so we don't have to revisit this later.

Reply:
I added an optional reason option